seahorn/jobs/linked_list_push_front/aws_linked_list_push_front_harness.c

```diff
@@ -0,0 +1,61 @@
+
+#include "nondet.h"
+#include <aws/common/linked_list.h>
+#include <seahorn/seahorn.h>
+
+int main(void) {
+  /* data structure */
+  struct aws_linked_list list;
+  struct aws_linked_list_node to_add;
+  struct aws_linked_list_node node1;
+  struct aws_linked_list_node *nnode;
+
+  // -- tail; should be untouched during operation under verification
+  list.tail.next = NULL;
```

Comment:
would be nice to factor this craft out into a function, similar to what is done in `swap_contents`.
In `swap_contents`, the function creates a linked list of sizes 0, 1, 2, or more.
Here we create a linked list whose forward size is 0, or >0.
In pop_back, we need a similar function but that creates a linked list whose bwd size is 0 or >0.

Where fwd len is counted from head, and bwd len is counted from tail.